DataCollection/src/Services/Controllers/BirdSpeciesController.cs

```diff
@@ -204,7 +204,7 @@ public IHttpActionResult GetBirdSpeciesBySurveyTypeId(int surveyTypeId)
             {
                 var birds = _domainManager.GetBirdSpeciesBySurveyTypeId(surveyTypeId);
 
-                var models = birds.Select(x => Map(x));
+                var models = birds?.Select(x => Map(x));
 
                 return base.Ok(models);
```

Comment:
In this case, wouldn't it be better to return an List<BirdSpecies> containing nothing, instead of returning null?